frontend/src/components/explorer/mapViewer/MapSearch.vue

```diff
@@ -106,7 +106,7 @@ export default {
       this.currentSearchMatch = 0;
       this.searchInputClass = 'is-info';
     },
-    async search(term) {
+    async search(term, centerId) {
       this.$store.dispatch('maps/setIdsFound', []);
```

Comment:
Not introduced in this PR and probably nothing wrong, but I am a bit curious why this call is not preceded with the `await` keyword? As of now the code below will run even if `this.$store.dispatch` fails, is that desirable?

Reply:
These calls to `$store.dispatch` is a part of Vuex and hooks into what we have defined in the store modules. In this instance, it's a simple action or [setter](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/frontend/src/store/modules/maps.js#L94-L96) that updates a variable. In pseudo code this is like running: `store.maps.idsFound = []`.

When you see an `await` in front of these `store.dispatch` calls, it's usually because the module action needs to make a network request.